documentation/shared/Docker_Hub_Container.md

```diff
@@ -20,10 +20,8 @@ The purpose of this task is to create a docker container with your code availabl
 ![](attachments/429031435/428965910.png?height=250)
 
 -   Choose "Create Automated Build" and grant access to your github account. Choose the option again and choose the github repo where your code is stored and kick off a build of the docker container.
+-   Ensure that the jar file of your project is uploaded to GitHub in order for Docker to succeed in creating an automated build. Standard java .gitignore files will have removed your /build folder.
```

Comment:
Can you explain this bit. Why are we pushing jar files to gitHub. Should the process not package the .jar file itself and then create the image based on that?

Reply:
I may have done this incorrectly. In my dockerfile I had: 

```
FROM openjdk:8-jdk-alpine
RUN mkdir =p /app
ARG JAR_FILE=build/libs/capcodigitalengineeringcourse-0.1.0.jar
ADD ${JAR_FILE} /app/app.jar
ENTRYPOINT...
```

which is slightly different to the bootJar method. I can remove this section.

Reply:
Let's remove it then :)

Reply:
I've now removed this section and made a signed commit for the update.